### PR TITLE
Fix genesis flags

### PR DESCRIPTION
**Pull request by iulianpascalau**
## Reasoning behind the pull request
- the nodes can not start & work properly if all enable epoch flags are set to 0
  
## Proposed changes
- fixed the genesis process to take into account 0-value flags
Related https://github.com/multiversx/mx-chain-core-go/pull/288

## Testing procedure
- 
- 
- 

## Pre-requisites

Based on the [Contributing Guidelines](https://github.com/multiversx/mx-chain-go/blob/master/.github/CONTRIBUTING.md#branches-management) the PR author and the reviewers must check the following requirements are met:
- was the PR targeted to the correct branch?
- if this is a larger feature that probably needs more than one PR, is there a `feat` branch created?
- if this is a `feat` branch merging, do all satellite projects have a proper tag inside `go.mod`?

### CODE REVIEW

File: factory/processing/processComponents.go [hunk 1]
```diff
@@ -240,6 +240,7 @@ func NewProcessComponentsFactory(args ProcessComponentsFactoryArgs) (*processCom
 		txExecutionOrderHandler: args.TxExecutionOrderHandler,
 		genesisNonce:            args.GenesisNonce,
 		genesisRound:            args.GenesisRound,
+		roundConfig:             args.RoundConfig,
```
Comment (raduchis) [inline factory/processing/processComponents.go:243]: :+1:

**Review by raduchis — APPROVED** [review]

File: node/chainSimulator/components/processComponents.go [hunk 2]
```diff
@@ -42,6 +42,7 @@ type ArgsProcessComponentsHolder struct {
 	NodesCoordinator     nodesCoordinator.NodesCoordinator
 
 	EpochConfig              config.EpochConfig
+	RoundConfig              config.RoundConfig
```
Comment (miiu96) [inline node/chainSimulator/components/processComponents.go:45]: can you remove from  file
`../confgs/configs.go` line 122  (`configs.EpochConfig.EnableEpochs.StakingV2EnableEpoch = args.InitialEpoch + 1`)

if the problem is fixed tests should pass without this line

Comment (iulianpascalau) [reply node/chainSimulator/components/processComponents.go:45]: done, all tests pass without that line

**Review by miiu96 — APPROVED** [review]

**Review by raduchis — APPROVED** [review]

**Review by gabi-vuls — APPROVED** [review]
Normal allin test: v1.6.15-dev-config-eb2e06c06d -> fix-genesis-flags-333c4f2461

--- Specific errors ---

block hash does not match 2605
wrong nonce in block 1010
miniblocks does not match 0
num miniblocks does not match 0
miniblock hash does not match 0
block bodies does not match 0
receipts hash missmatch 0

/------/


--- Statistics ---

Nr. of all ERRORS:  9
Nr. of all WARNS:  9367
Nr. of new ERRORS:  9
Nr. of new WARNS:  81
Nr. of PANICS:  0

/------/

--- ERRORS ---

/------/

**Pull request merged**